src/App.js

```diff
@@ -14,16 +14,16 @@ import React, { useState, useEffect } from 'react';
 function App() {
   const [products, setProducts] = useState([]);
 
-  const fetchProducts = async () => {
-    const { data } = await commerce.products.list();
-
-    setProducts(data);
-  };
-
-  useEffect(() => {
-    fetchProducts();
-    
-  }, []);
+  // const fetchProducts = async () => {
```

Comment:
If the Products page needs products, it's probably best for it to fetch it there and not be passed in from this page - otherwise you're fetching the products for every page on the site even if only one sub-page needs it